app/templates/explore.hbs

```diff
@@ -35,6 +35,12 @@
           <a role="button" {{action 'clearFilter' 'end_date'}}><i class="icon close"></i></a>
         </div>
       {{/if}}
+      {{#if filters.cfs}}
+        <div class="ui mini label">
+          {{if (eq filters.cfs 'true') 'Open' 'Closed'}}
```

Comment:
no need to explicitly compare with true.

Reply:
doing `{{if filters.cfs 'Open' 'Closed'}}` results in only `Open` being set, even on selecting `Closed`.

Reply:
@Anupam-dagar then it's probably because `filters.cfs` is a string and not a boolean. Ensure it is a boolean.

Reply:
@niranjan94 I am setting it as boolean only. `this.set('cfs', cfs === this.cfs ? null : (cfs === 'open' ? true : false));`
Also if `filters.cfs` is boolean, then shouldn't the `if` block must not get executed when `closed` is selected since it will become `if false`.

Reply:
@Anupam-dagar check the data type in the model definition (if one exists). Also check if it is being assigned as a non-boolean anywhere else. Because the only explanation for the behaviour you talk about is if it is a non-boolean.

Reply:
@niranjan94 I refactored my code, it was being set as boolean but I interpreted it wrong in the router. A little weird thing is happening. Please take a look at the screenshot.
![fossasia](https://user-images.githubusercontent.com/21174572/59951073-6c541b00-9495-11e9-9abb-18a70d8ed66e.png)

Before the first click, no filter is set for cfs. I click on `Open`, the cfs value passed in action as parameter is `true`. Since `cfs` variable for request isn't set, `this.cfs` is `null`. The type here is boolean and Object as expected. I set the variable `cfs` to the param value. Now the types are boolean and boolean as expected.

Now I do the second click on the same filter to unset it, before the `this.set` line, the cfs param and `this.cfs` are both `true` but the types are `boolean` and `string` which should have been `boolean` and `boolean` as `this.cfs` was set to a `boolean` which can be seen from the previous console log. Now after the `this.set` statement, again the cfs is set to `true` which should have been `null` but since the type mismatched it is set to `true`. Finally the types now are both `boolean`.

For the third click, the type for `this.cfs` is boolean and hence I can unset the filter. I am not able to understand how the type is getting changed from `boolean` to `string`.

Also I am still unsure for this code block
```
{{#if filters.cfs}}
        <div class="ui mini label">
          {{if filters.cfs 'Open' 'Closed'}}
          <a role="button" {{action 'clearFilter' 'cfs'}}><i class="icon close"></i></a>
        </div>
{{/if}}
```
I tried after setting to boolean too. I am concerned about the case when `filters.cfs` is `false`, in that case will the `if` block get executed since the value is false. If it doesn't get executed then we won't be able to display the 'Closed' label.

Reply:
@shreyanshdwivedi @uds5501 @mrsaicharan1 can you please take a look here.